packages/nuxt/src/app/plugins/revive-payload.client.ts

```diff
@@ -49,7 +49,6 @@ export default defineNuxtPlugin({
       definePayloadReviver(reviver, revivers[reviver as keyof typeof revivers])
     }
     Object.assign(nuxtApp.payload, await nuxtApp.runWithContext(getNuxtClientPayload))
-    // For backwards compatibility - TODO: remove later
-    window.__NUXT__ = nuxtApp.payload
+    delete window.__NUXT__
```

Comment:
I wonder: Is there a specific reason why we wanted to remove it on the first place? 🤔

Reply:
@manniL I guess to hide the explicit bucket where has useful data. In my case, I pass token for integrations and there is no problem to be explicit, but window.__NUXT__ is very chamative

Reply:
to be clear, this is not about security - it is essential to treat all client side code as non privileged and fully accessible by end user.

In this case `window.useNuxtApp().payload` will return the _same information_.